### DISCUSSION TITLE
Chezmoi with Dropbox as sync backend

### DISCUSSION THREAD

**Discussion by halloleo** [twpayne/chezmoi#3797] [Q&A] [answered]
Looking at chezmoi. :)

However I want to do the machine-to-machine sync via a Dropbox dir.

So I tried

    chezmoi init -S ~/Dropbox/sync/chezmoi
    chezmoi add ~/myfile
    chezmoi apply

And on the other machine:

    chezmoi init -S ~/Dropbox/sync/chezmoi
    chezmoi apply

Now I get error "chezmoi: stat ~/.local/share/chezmoi: no such file or directory". 

What do I do wrong?

**Comment by twpayne** [comment, marked as answer]
You need to pass the `-S` option to `chezmoi add` and `chezmoi apply`, or set the `sourceDir` variable in your config file.

**Reply by halloleo** [reply]
Thanks @twpayne for chiming in! 😄

So that I understand it correctly: In order to use chezmoi on two machines via `~/Dropbox/sync/chezmoi`, I have create manually _on both machines_ the same config file `~/.config/chezmoi/chezmoi.toml` with the content:

    # Chezmoi config via Dropbox
    sourceDir = "~/Dropbox/sync/chezmoi"

Then I do `chezmoi init` on both machines and then I can sync _other_ config files (like `.bashrc`) via the normal add-apply-apply procedure, correct?

**Reply by bradenhilton** [reply]
@halloleo You can create a [config file template](https://www.chezmoi.io/reference/special-files-and-directories/chezmoi-format-tmpl/) in `~/Dropbox/sync/chezmoi`, which [`chezmoi init`](https://www.chezmoi.io/reference/commands/init/) will use to create a config automatically, though this approach might still require an initial `-S` flag until the config file is created.
